gix-submodule/tests/file/baseline.rs

```diff
@@ -61,7 +61,7 @@ fn common_values_and_names_by_path() -> crate::Result {
 
 fn module_files() -> impl Iterator<Item = (PathBuf, PathBuf)> {
     let dir = gix_testtools::scripted_fixture_read_only("basic.sh").expect("valid fixture");
-    gix_features::fs::walkdir_sorted_new(&dir, Parallelism::Serial, false)
+    gix_features::fs::walkdir_sorted_new(&dir, Parallelism::Serial, usize::MAX, false)
```

Comment:
This should go into a separate commit right after the one changing `gix-ref` conventionally saying something like `adapt to changes in gix-features`.